### PR TITLE
Added Matlab courses in hindi

### PR DESCRIPTION
**Pull request by PuneetTri**
## What does this PR do?
Added courses for Matlab, both a one-shot course and a self-paced playlist course.

### Why is this valuable (or not)?
A great resource for learning Matlab, recommended by a lot of college professors

### How do we know it's really free?
It's available on YouTube!

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -361,6 +362,12 @@
 * [Statistics And Probability](https://www.youtube.com/playlist?list=PLU6SqdYcYsfLRq3tu-g_hvkHDcorrtcBK) - Gajendra Purohit
 
 
+### Matlab
+
+* [Matlab Complete Course](https://www.youtube.com/watch?v=iS5J4TlLSEM) - Armughan Ali
+* [Matlab for Beginners (Playlist)](https://www.youtube.com/playlist?list=PLjVLYmrlmjGfdIlwG649bdzVHM4iLbD_H) - WsCube Tech, `tch.:` VK Singh
```
Comment (shrn01) [inline courses/free-courses-hi.md:368]:
```suggestion
* [MATLAB Complete Tutorial in Hindi](https://www.youtube.com/playlist?list=PLjVLYmrlmjGfdIlwG649bdzVHM4iLbD_H) - WsCube Tech, `tch.:` VK Singh
```
Do not change the resource name

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: courses/free-courses-hi.md [hunk 2]
```diff
@@ -361,6 +362,12 @@
 * [Statistics And Probability](https://www.youtube.com/playlist?list=PLU6SqdYcYsfLRq3tu-g_hvkHDcorrtcBK) - Gajendra Purohit
 
 
+### Matlab
+
+* [Matlab Complete Course](https://www.youtube.com/watch?v=iS5J4TlLSEM) - Armughan Ali
+* [MATLAB Complete Tutorial in Hindi](https://www.youtube.com/playlist?list=PLjVLYmrlmjGfdIlwG649bdzVHM4iLbD_H) - WsCube Tech, `tch.:` VK Singh
```
Comment (eshellman) [inline courses/free-courses-hi.md:368]:
```suggestion
* [MATLAB Complete Tutorial in Hindi](https://www.youtube.com/playlist?list=PLjVLYmrlmjGfdIlwG649bdzVHM4iLbD_H) - WsCube Tech
``` just use attribution on the playlist page

File: courses/free-courses-hi.md [hunk 3]
```diff
@@ -361,6 +362,13 @@
 * [Statistics And Probability](https://www.youtube.com/playlist?list=PLU6SqdYcYsfLRq3tu-g_hvkHDcorrtcBK) - Gajendra Purohit
 
 
+### Matlab
+
+* [Matlab Complete Course](https://www.youtube.com/watch?v=iS5J4TlLSEM) - Armughan Ali
+* [MATLAB Complete Tutorial in Hindi](https://www.youtube.com/playlist?list=PLjVLYmrlmjGfdIlwG649bdzVHM4iLbD_H) - WsCube Tech
+``` just use attribution on the playlist page
```
Comment (shrn01) [inline courses/free-courses-hi.md:369]: Remove this line 
```suggestion
```

**Comment by PuneetTri** [comment]
@shrn01 I'm sorry I thought that was just a comment and can be left in. But I've now committed your suggestions

**Pull request merged**